Muvr/Entities/MRManagedExerciseSession+JSON.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
```

Comment:
I thought all that we wanted to do now was to upload a CSV file with the data; though this is actually OK.